v2/emailpassword/nextjs/app-directory/server-components-requests.mdx

```diff
@@ -66,7 +66,7 @@ export async function HomePage() {
         if (hasInvalidClaims) {
             return <SessionAuthForNextJS />;
         } else {
-            return <TryRefreshComponent />;
+            return <TryRefreshComponent key={Date.now()} />;
```

Comment:
add comment explaining why this is needed everywhere

Reply:
done